conf.d/python.d/apache_cache.conf

```diff
@@ -0,0 +1,4 @@
+# Example configuration of apache_cache.chart.py
+# YAML format
+
+path: "/var/log/apache2/cache.log"
```

Comment:
Same as with `hddtemp.chart.py`.
Also as I said earlier **every** module supports jobs because they are implemented in `python.d.plugin`.